learning_journal/learning_journal/templates/404.jinja2

```diff
@@ -0,0 +1,8 @@
+{% extends "layout.jinja2" %}
+
+{% block content %}
+<div class="content">
+  <h1><span class="font-semi-bold">Pyramid</span> <span class="smaller">Alchemy scaffold</span></h1>
+  <p class="lead"><span class="font-semi-bold">404</span> Page Not Found</p>
+</div>
+{% endblock content %}
```

Comment:
You might consider updating this to fit the fact that you've created a learning journal.  Perhaps it could print something like "The entry you were looking for cannot be found" or something like that.  Or maybe just have a cute picture of a confused-looking cat.